src/atoms/LinkBarSearchField.jsx

```diff
@@ -1,45 +1,62 @@
-import React, { Fragment } from 'react';
+/* eslint-disable jsx-a11y/label-has-for, jsx-a11y/label-has-associated-control */
+import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 
-import { getColor, getVariable } from '../utils';
+import { getColor, getVariable, linkBarElementSizes } from '../utils';
 import { LinkBarElementBase } from './LinkBarElementBase';
-import { FontIcon } from './FontIcon';
+import { SvgIcon } from './SvgIcon';
+
+const AbsoluteIcon = styled(SvgIcon)`
+	${linkBarElementSizes}
 
-const FontIconBase = LinkBarElementBase.withComponent(FontIcon);
-const AbsoluteFontIcon = styled(FontIconBase)`
 	${(props) => {
-		const { size } = props;
+		const { fullWidth, size } = props;
 		const horizontalBase = getVariable('horizontalBase')(props);
+		const uiSmallLineHeight = getVariable('uiSmallLineHeight')(props);
+		const uiRegularLineHeight = getVariable('uiRegularLineHeight')(props);
 
 		const smallMarginFactor = size === 'small' ? 1/2 : 1;
-		const horizontalMargin = `calc(${smallMarginFactor} * ${horizontalBase})`;
+		const fullWidthFactor = 3/2;
```

Comment:
fullWidthFactor will never change?

Reply:
don't think it will change, no. This is used in calculating the icons position - not the width of the search field.